File: Python/Q3_Encapsulamento.py
from typing import Any


class ContaBancaria:
    def __init__(self,saldo,titular):
        self.__saldo=saldo
        self.titular=titular
    def depositar(self,dinheiro):
        self.__saldo+=dinheiro
    def sacar(self,dinheiro):
        self.__saldo-=dinheiro
    def get_Saldo(self):
        return self.__saldo
conta = ContaBancaria(100,"João")
print(conta.get_Saldo())
conta.depositar(200)
print(conta.get_Saldo())
conta.sacar(50)
print(conta.get_Saldo())

File: go/Q3_Encapsulamento/Q3_Encapsulamento.go
package main

import "fmt"

type ContaBancaria struct {
	titular string
	saldo float64
}
func (c *ContaBancaria) Depositar(dinheiro float64) {
	c.saldo+=dinheiro
}
func (c *ContaBancaria) Sacar(dinheiro float64) {
	c.saldo-=dinheiro
}
func main(){
	conta := ContaBancaria{titular: "João",saldo: 100}
	conta.Depositar(200)
	conta.Sacar(50)
	fmt.Printf("%f",conta.saldo)
}